hdk/assembly/syntax.py

```diff
@@ -96,6 +96,21 @@ class CInstruction:
     dest: str | None = None
     jump: str | None = None
 
+    @property
+    def is_m_dest(self) -> bool:
+        """Check if memory[a resistor] is dest."""
```

Comment:
It is a property. So, the docstring for it should be as if it is a noun (not a verb) and should not start from checks/check etc. For example: True if RAM (addressed by A-register) is the destination of the computation.